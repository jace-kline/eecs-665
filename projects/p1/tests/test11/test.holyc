"Testing empty char literal error" '
"Testing empty char escape error" '\
"Testing inline comment" # This is a comment
# This is another comment
(true) {123}