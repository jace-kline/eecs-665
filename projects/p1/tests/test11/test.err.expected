FATAL [1,36]: Empty character literal
FATAL [2,35]: Empty escape sequence in character literal
